lib/DateTime/Format/Strptime/Types.pm

```diff
@@ -30,6 +30,7 @@ coerce(
 );
 
 object_isa_type( 'TimeZone', class => 'DateTime::TimeZone' );
+object_isa_type( 'DateTime', class => 'DateTime' );
```

Comment:
I also realized that you need to add `use DateTime` to this module, or else the compilation tests fail.